docs/PERSISTENCE.md

```diff
@@ -0,0 +1,57 @@
+[-UP-](./README.md)
+
+
+# Plog
+The persistence data model provided/used by Chogori Project is called Plog. It is an interface for file-append-like storage services, such as PLOG or Azure AppendBlock. In our current design, this Plog is an append-only struct. The only operation that allowed to change the Plog is to append new data after the current blocks. 
+
+## Term
+- Plog Client: Process all the operation requests regarding the Plog. It communicates with Plog Servers. It supports 4 operations: create, append, read, and seal.
+- Plog Server: Communicate with Plog Client and process all the requests. It reads/appends data from/to the storage media.
+- Persistence Group: A Persistence Group consists of 3 Plog Servers. When a Plog Client will create/append/seal a plog, it will always communicate with all the Plog Servers in the same Persistence Group simultaneously in order to provide redundancy.
```

Comment:
```suggestion
- Persistence Group: A Persistence Group consists of 3 Plog Servers. When a Plog Client creates/appends/seals a plog, it will always communicate with all the Plog Servers in the same Persistence Group simultaneously in order to provide redundancy.
```